site/gatsby-site/gatsby-config.js

```diff
@@ -248,7 +248,4 @@ module.exports = {
   },
   plugins: plugins,
   trailingSlash: `always`,
-  flags: {
```

Comment:
To allow the API route like `/cite/[id]` we have to remove that. Otherwise we have this error: 
<img width="785" alt="image" src="https://user-images.githubusercontent.com/6564809/228935045-b50d3d11-8344-4498-90b8-4ea93ec00b24.png">

Reply:
This is a dev-only setting. 

We should keep it enabled; otherwise, we'll be back to pushing code with hydration errors 💀 

It might be possible to avoid that error if you visit a page that isn't using the client-side route first, let the client-only shell build, and then go to `/cite/[id]`. If that doesn't work, we should only disable it locally while working on that page.

Reply:
done!